Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -155,10 +161,19 @@ public override LayerMask[] PrioritizedLayerMasksOverride
         [Tooltip("Whether to ignore colliders that may be near the pointer, but not actually in the visual FOV. This can prevent accidental grabs, and will allow hand rays to turn on when you may be near a grabbable but cannot see it. Visual FOV is defined by cone centered about display center, radius equal to half display height.")]
         private bool ignoreCollidersNotInFOV = true;
 
+        [SerializeField]
+        [Tooltip("Location on the hand where a grasp is triggered (apllies to IMixedRealityHand only)")]
+        private GraspPointPlacement graspPointPlacement = GraspPointPlacement.BetweenFingerAndThumb;
+
+        /// <summary>
+        /// Location on the hand where a grasp is triggered (apllies to IMixedRealityHand only)
+        /// </summary>
+        public GraspPointPlacement GraspPointPlacement => graspPointPlacement;
```

Comment:
If this doesn't have a public setter, is it useful to have a public getter? Or could this just be an implementation detail?

Reply:
And if this is removed, I believe the `GraspPointPlacement` enum could be made `private` as well and keep some of this off the public API surface.